R/doc-params.R

```diff
@@ -1,31 +1,117 @@
+#' Argument type: data-masking
+#'
+#' @description
+#' This page describes the `<data-masking>` argument modifier which
+#' indicates that the argument uses **data masking**, a sub-type of
+#' tidy evaluation. If you've never heard of tidy evaluation before,
+#' start by reading `vignette("programming")` and
+#' <https://r4ds.hadley.nz/functions.html#data-frame-functions>.
+#'
+#' # Key techniques
+#'
+#' *  If you want the user to supply the variable (or function of variables)
```

Comment:
It's a bit unclear what is meant by "function of variables". Maybe just omit these parentheses?